packages/cli/src/ui/commands/memoryCommand.test.ts

```diff
@@ -230,19 +220,7 @@ describe('memoryCommand', () => {
         expect.any(Number),
       );
 
-      expect(mockLoadHierarchicalGeminiMemory).toHaveBeenCalledOnce();
```

Comment:
Note that these parts of the test are not gone, they are a part of the memoryDiscovery.test.ts file now (which tests the behavior of refreshServerHierarchicalMemory)